apps/webapp/src/app/login/login-email.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, TextField } from '@mui/material';
+import styled from '@emotion/styled';
+import { FC, useState } from 'react';
+
+const disabled = true;
+export const LoginEmail: FC = () => {
+  const [email, setEmail] = useState<string>();
+  const [password, setPassword] = useState<string>();
+  return (
+    <Panel>
+      <StyledTextField
+        disabled={disabled}
+        size="small"
+        autoFocus
+        value={email ?? ''}
+        onChange={(change) => setEmail(change.target.value)}
+        label={'Email'}
+      />
+      <StyledTextField
+        disabled={disabled}
+        size="small"
+        value={password ?? ''}
+        onChange={(change) => setPassword(change.target.value)}
+        type="password"
+        label={'Hasło'}
+      />
+      <Button
+        disabled={disabled}
+        onClick={() => {
+          if (email && password) {
+            alert('Not yet implemented, use firebase auth');
```

Comment:
na to jest jakieś issue?

Reply:
niet, trochę zakładam, że to się łapie jako część zadania #11

Reply:
pytanie czy w ogóle chcemy mail? Krysia mnie przekonuje, że "wszyscy mają gmail" a jak nie mają to "trudno" :)

@drolewski @ksolawa

Reply:
@ksolawa ma rację w sumie.
Wydzielmy na to issue z niskim prio. Można zrobić, ale nie jest niezbędne do życia. A trzebaby też dorobić wtedy mechanizm potwierdzania tego emaila pewnie itp.

Reply:
nie było tematu, logowanie, rejestrowanie i reset hasła do maili dodany.